server/api/impl/upstream/upawslambda/upstream.go

```diff
@@ -100,3 +118,21 @@ func callToInvocationPayload(call *apps.CallRequest) ([]byte, error) {
 
 	return payload, nil
 }
+
+func match(callPath string, app *apps.App) (string, error) {
+	matchedName := ""
+	matchedPath := ""
+	for _, f := range app.Functions {
+		if strings.HasPrefix(callPath, f.Path) {
+			if len(f.Path) > len(matchedPath) {
+				var err error
+				matchedPath = f.Path
+				matchedName, err = awsclient.GenerateLambdaName(app.AppID, app.Version, f.Name)
+				if err != nil {
+					return "", err
+				}
+			}
+		}
+	}
+	return matchedName, nil
```

Comment:
Should we return `utils.ErrNotFound` if no match is found?

Reply:
No, we take the longest match; if you have a `/` it'll be returned as the last resort